File: core/src/androidTest/java/com/emarsys/core/testUtil/TimeoutUtils.java
package com.emarsys.core.testUtil;

import android.support.annotation.NonNull;
import android.support.test.rule.DisableOnAndroidDebug;

import org.junit.rules.TestRule;
import org.junit.rules.Timeout;

public class TimeoutUtils {

    private TimeoutUtils() {
    }

    @NonNull
    public static TestRule getTimeoutRule() {
        return new DisableOnAndroidDebug(Timeout.seconds(30));
    }

}

File: testUtils/src/main/java/com/emarsys/test/util/TimeoutUtils.kt
package com.emarsys.test.util

import android.support.test.rule.DisableOnAndroidDebug

import org.junit.rules.Timeout

object TimeoutUtils {

    @JvmStatic
    val timeoutRule
        get () = DisableOnAndroidDebug(Timeout.seconds(30))

}
